precompile/contracts/nativeminter/config_test.go

```diff
@@ -17,29 +17,30 @@ import (
 func TestVerify(t *testing.T) {
 	admins := []common.Address{allowlist.TestAdminAddr}
 	enableds := []common.Address{allowlist.TestEnabledAddr}
+	managers := []common.Address{allowlist.TestManagerAddr}
 	tests := map[string]testutils.ConfigVerifyTest{
 		"invalid allow list config in native minter allowlist": {
-			Config:        NewConfig(utils.NewUint64(3), admins, admins, nil),
+			Config:        NewConfig(utils.NewUint64(3), admins, admins, nil, nil),
 			ExpectedError: "cannot set address",
 		},
 		"duplicate admins in config in native minter allowlist": {
-			Config:        NewConfig(utils.NewUint64(3), append(admins, admins[0]), enableds, nil),
+			Config:        NewConfig(utils.NewUint64(3), append(admins, admins[0]), enableds, managers, nil),
 			ExpectedError: "duplicate address",
 		},
 		"duplicate enableds in config in native minter allowlist": {
-			Config:        NewConfig(utils.NewUint64(3), admins, append(enableds, enableds[0]), nil),
+			Config:        NewConfig(utils.NewUint64(3), admins, append(enableds, enableds[0]), managers, nil),
```

Comment:
did we add a UT to check duplicate addresses in the manager list with the other lists?

Reply:
yes we have added them in `test_allowlist_config.go`